pkgs/by-name/fe/fex/package.nix

```diff
@@ -11,19 +11,98 @@
   xxHash,
   fmt,
   nasm,
+  buildEnv,
+  runCommand,
+  writeText,
+  pkgsCross,
+  libclang,
+  libllvm,
+  alsa-lib,
+  libdrm,
+  libGL,
+  wayland,
+  xorg,
 }:
 
+let
+  pkgsCross32 = pkgsCross.gnu32;
+  pkgsCross64 = pkgsCross.gnu64;
+  devRootFSBase = buildEnv {
+    name = "fex-dev-rootfs";
+    paths = [
+      # later stdenv seems to have shuffled around the headers
+      # pkgsCross64.stdenv.cc.libc_dev
+      # pkgsCross32.stdenv.cc.libc_dev
+      # pkgsCross64.stdenv.cc.cc
+      # pkgsCross32.stdenv.cc.cc
+      pkgsCross64.gcc13Stdenv.cc.libc_dev
+      pkgsCross32.gcc13Stdenv.cc.libc_dev
+      pkgsCross64.gcc13Stdenv.cc.cc
+      pkgsCross32.gcc13Stdenv.cc.cc
+
+      alsa-lib.dev
+      libdrm.dev
+      libGL.dev
+      wayland.dev
+      xorg.libX11.dev
+      xorg.libxcb.dev
+      xorg.libXrandr.dev
+      xorg.libXrender.dev
+      xorg.xorgproto
+    ];
+    ignoreCollisions = true;
+    pathsToLink = [
+      "/include"
+      "/lib"
+    ];
+  };
+
+  devRootFS = runCommand "fex-dev-rootfs2" {} ''
+    mkdir -p $out/usr
+    ln -s ${devRootFSBase}/* $out/
+    ln -s ${devRootFSBase}/include $out/usr/
+  '';
+
+  toolchain32 = writeText "toolchain_nix_x86_32.txt" "
```

Comment:
Sorry, just small nitpick, don't have the time yet to do a full review:
I think we should use `''...''` strings here. You can escape `${`s by writing `''${`.
See: https://nix.dev/manual/nix/2.29/language/string-literals.html